protocol/contracts/leaser/src/leaser.rs

```diff
@@ -121,28 +121,36 @@ where
     })
 }
 
-pub(super) fn try_close_protocol<ProtocolsRegistryLoader>(
-    storage: &dyn Storage,
+pub(super) fn try_close_protocol<ProtocolsRegistryLoader, MsgFactory>(
+    storage: &mut dyn Storage,
+    new_lease_code: Code,
+    max_leases: MaxLeases,
```

Comment:
Why is this a parameter to the function? As we discussed, the case where not all leases are migrated is invalid, thus rendering this parameter redundant.

Reply:
for consistency with the other functions
the parameter receives Uxx::MAX otherwise.